android/app/src/main/java/com/on/turip/ui/search/detail/SearchDetailActivity.kt

```diff
@@ -27,9 +49,89 @@ class SearchDetailActivity : BaseActivity<SearchDetailViewModel, ActivitySearchD
             this,
             object : OnBackPressedCallback(true) {
                 override fun handleOnBackPressed() {
-                    finish()
+                    if (turipWebChromeClient.isFullScreen()) {
+                        turipWebChromeClient.onHideCustomView()
+                    } else if (webView.canGoBack()) {
+                        webView.goBack()
+                    } else {
+                        finish()
+                    }
                 }
             },
         )
     }
+
+    private fun setupBindings() {
+        webView = binding.wvSearchDetailVideo
+    }
+
+    private fun setupWebView() {
+        turipWebChromeClient =
+            TuripWebChromeClient(
+                fullScreenView = binding.flSearchDetailVideoFullscreen,
+                onEnterFullScreen = ::enableFullscreen,
+                onExitFullScreen = ::disableFullscreen,
+            )
+
+        webView.applyVideoSettings()
+        webView.webChromeClient = turipWebChromeClient
+        webView.webViewClient = TuripWebViewClient(binding.pbSearchDetailVideo)
+        webView.addJavascriptInterface(
+            // TODO 생성자에 url의 일부 추출 필요, WebViewUtils에 구현한 String.extractVideoId() 사용 예정
+            WebViewVideoBridge("") { showWebViewErrorView() },
+            BRIDGE_NAME_IN_JS_FILE,
+        )
+        webView.loadUrl(LOAD_URL_FILE_PATH)
+    }
+
+    private fun enableFullscreen() {
+        WindowCompat.setDecorFitsSystemWindows(this.window, false)
+        WindowInsetsControllerCompat(window, window.decorView).apply {
+            hide(WindowInsetsCompat.Type.systemBars())
+            systemBarsBehavior =
+                WindowInsetsControllerCompat.BEHAVIOR_SHOW_TRANSIENT_BARS_BY_SWIPE
+        }
+
+        binding.tbSearchDetail.visibility = View.GONE
+        binding.nsvSearchDetail.visibility = View.GONE
+
+        requestedOrientation = ActivityInfo.SCREEN_ORIENTATION_LANDSCAPE
+    }
+
+    private fun disableFullscreen() {
```

Comment:
c: 
```suggestion
    private fun disableFullScreen() {
```
풀 / 스크린 두 단어 일 것 같은 느낌스 .. ?